install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml

```diff
@@ -25,6 +25,10 @@ spec:
       configuration is failing.
     name: Degraded
     type: string
+  - JSONPath: .spec.paused
```

Comment:
please remove this - this extends the output to be way too large and we can deal with this in a different way. once this is reverted we can merge, we will add further changes as a follow-on pr.

Reply:
just reverted that change